### ISSUE TITLE
Accessibility Issue

### ISSUE THREAD

**Issue by sabbCodes** [mozilla/perfcompare#352]
I noticed that the compare revision button was lacking in accessibility as it doesn't have an aria-label.
I fixed it already;
https://github.com/mozilla/perfcompare/pull/337
